inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -601,5 +611,62 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::ma
     return res;
 }
 
+InferenceEngine::IExecutableNetworkInternal::Ptr Engine::ImportNetwork(
+                                            const std::string &modelFileName,
+                                            const std::map<std::string, std::string> &config) {
+    std::fstream inputStream(modelFileName, std::ios::in | std::ios::binary);
+    if (inputStream.fail()) {
+        IE_THROW() << "Cannot open file to import model: " << modelFileName;
+    }
+    return ImportNetwork(inputStream, config);
+}
+
+InferenceEngine::IExecutableNetworkInternal::Ptr Engine::ImportNetwork(std::istream& networkModel,
+                                            const std::map<std::string, std::string>& config) {
+    OV_ITT_SCOPE_CHAIN(FIRST_INFERENCE, taskChain, itt::domains::MKLDNN_LT, "ImportNetwork", "ReadFromFile");
+
+    using namespace ngraph::pass;
+
+    std::string xmlString;
+    InferenceEngine::Blob::Ptr dataBlob;
+
+    StreamSerialize::DataHeader hdr = {};
+    networkModel.read(reinterpret_cast<char*>(&hdr), sizeof hdr);
+
+    // read blob content
+    networkModel.seekg(hdr.consts_offset);
+    if (hdr.consts_size) {
+        dataBlob = InferenceEngine::make_shared_blob<std::uint8_t>(
+            InferenceEngine::TensorDesc(InferenceEngine::Precision::U8, {hdr.consts_size}, InferenceEngine::Layout::C));
+        dataBlob->allocate();
+        networkModel.read(dataBlob->buffer(), hdr.consts_size);
+    }
+
+    // read XML content
+    networkModel.seekg(hdr.model_offset);
+    xmlString.resize(hdr.model_size);
+    networkModel.read(const_cast<char*>(xmlString.c_str()), hdr.model_size);
+
+    OV_ITT_SCOPE_SKIP(FIRST_INFERENCE, taskChain);
+
+    CNNNetwork cnnnetwork = GetCore()->ReadNetwork(xmlString, std::move(dataBlob), extensionManager->Extensions());
+
+    // TODO: Clarify the behavior of SetConfig method. Skip eng_config or not?
+    Config conf = engConfig;
```

Comment:
Can we remove this todo? Behavior is correct

Reply:
removed